plugins/woocommerce/includes/abstracts/abstract-wc-payment-gateway.php

```diff
@@ -324,8 +324,7 @@ public function get_description() {
 	 * @return string
 	 */
 	public function get_icon() {
-
-		$icon = $this->icon ? '<img src="' . WC_HTTPS::force_https_url( $this->icon ) . '" alt="' . esc_attr( $this->get_title() ) . '" />' : '';
+		$icon = $this->icon ? '<img src="' . esc_attr( WC_HTTPS::force_https_url( $this->icon ) ) . '" alt="' . esc_attr( $this->get_title() ) . '" />' : '';
```

Comment:
`esc_url()` is more appropriate here, to ensure we don't send malicious IMG URLs

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/48432/commits/8b38bb1142f8e6156580699c2754973b363a15ed